android/build.gradle

```diff
@@ -3,6 +3,10 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
```

Comment:
I believe that with the gradle version used, we could just add `google()` here. Could you check it out? More info [here](https://developer.android.com/studio/build/dependencies#google-maven)